.gitignore

```diff
@@ -1,4 +1,12 @@
 /result*
 \#*\#
 *.*~
-*.bak
\ No newline at end of file
+*.bak
+
+## Editor artifacts and configuration files
+TAGS
+.dir-locals.el
+*.md.temp
```

Comment:
Slight note but IMO these (as well as any OS-related files) shouldn't part of a project `.gitignore` for these only regard your own setup. 

Instead, I'd recommend using a global `.gitignore` for your shell using the `core.excludesfile` configuration option for git.

**~/.gitconfig**
```gitconfig
[core]
  excludesfile = /home/ktorz/.gitignore # Always ignore these files, in every git project
```

which in my case, points to the following:

**~/.gitignore**
```gitignore
### OSX Bullshit ###
.DS_Store
Thumbs.db
*.orig

### Tags File ###
tags

### Git Worktrees ###
.worktrees

### Vim ###
[._]*.s[a-w][a-z]
[._]s[a-w][a-z]
*.un~
Session.vim
.netrwhist
*~

### Stack ###
.stack-work-* # extra working trees manually created sometimes (e.g. .stack-work-profiling)
```

Reply:
That's a good idea, however I'm worried that this would rely on each developer setting up her own `~/.gitconfig` properly.

Reply:
Yes. It does :)